provider/pkg/gen/discovery.go

```diff
@@ -142,6 +155,19 @@ func findResourcesImpl(docName, parentName string, rest map[string]discovery.Res
 		}
 
 		if strings.Contains(strings.ToLower(name), "operation") {
+			if getMethod != nil {
+				operationGetPath := getMethod.FlatPath
+				if operationGetPath == "" {
+					operationGetPath = getMethod.Path
+				}
+				if override, has := resourceNameByPathOverrides[fmt.Sprintf("%s:%s", filepath.Base(docName),
+					operationGetPath)]; has {
+					if override == "" {
+						continue
+					}
+					name = override
```

Comment:
```suggestion
					if override != "" {
						name = override
					}
```

Reply:
I have added a few more comments here which hopefully help. The intent with an empty string value is to ignore the specified resource/operation.